### PR TITLE
Add smoke tests doc

### PR DESCRIPTION
**Pull request by bitoku**
### Which issue this PR addresses:

Fixes part of https://issues.redhat.com/browse/ARO-6771

### What this PR does / why we need it:

This PR adds document about how to run smoke tests.

### Test plan for issue:

doc fix, N/A

### Is there any documentation that needs to be updated for this PR?

doc fix, N/A

### How do you know this will function as expected in production? 

doc fix, N/A

### CODE REVIEW

**Comment by nwnt** [comment]
@bitoku please ignore the tests I executed here.  I just wanted to test if ci and e2e are working correctly.

File: docs/testing.md [hunk 1]
```diff
@@ -53,19 +53,34 @@ These steps can be acheived using commands below.  Look at the [e2e helper
...
+You can run the smoke tests by running the following command:
+
+```bash
+E2E_LABEL=smoke make test-e2e
+```
+
+If you want to run both e2e and smoke tests:
+
+```bash
+E2E_LABEL= make test-e2e
```
Comment (cadenmarchese) [inline docs/testing.md:71]: Is this required, or can we simplify?
```suggestion
make test-e2e
```

Comment (bitoku) [reply docs/testing.md:71]: It's required to disable default.

https://github.com/Azure/ARO-RP/blob/50939e1a381c0795bc13d2bd3d32c48ca39cfa40/Makefile#L6

**Review by kimorris27 — APPROVED** [review]

**Review by tiguelu — APPROVED** [review]

**Pull request merged**